r/src/tests/testthat/testMojo.R

```diff
@@ -218,4 +218,27 @@ test_that("test getModelSummary", {
   expect_equal(count, 4)
 })
 
+test_that("test MOJO contribution (SHAP) values", {
```

Comment:
This functionality is related to DAI Mojos. Can you move this test to dedicated file?

Reply:
A new file "testMojoPipeline.R" was created. Furthermore, build pipeline was updated to provided a license file needed to run DAI MOJOs.